spec/lib/reportsv2_spec.rb

```diff
@@ -237,7 +237,7 @@
       end
     end
 
-    context 'XLS reports' do
+    context 'XLS reports', :pro_account do
```

Comment:
Yes. I've gotten into some bad habits with this repo since this is (almost) a solo project.

Please merge the latest master branch into your branch. Once you update the PR, all the tests should pass. At that point I'll merge the PR.

I may rearrange some things but I really appreciate you solving this problem and I want you to have credit for the fix. Thanks!